app/controllers/service_requests/service_request_assignments_controller.rb

```diff
@@ -1,12 +1,8 @@
 class ServiceRequests::ServiceRequestAssignmentsController < ApplicationController
+  before_action :authenticate_customer!, :set_account
 
   def start_accepting
-    @account = current_customer.current_account
-    @service_request = @account.service_requests.find(params[:service_request_id])
-    @service_request.account_id = @account.id
-    @assignment = @service_request.assignments.find(params[:service_request_assignment_id])
-    @assignment.account_id = @account.id
-    @assignment.service_request_id = @service_request.id
+    find_service_request_and_assignment
```

Comment:
Why you call it in each method instead of using `before_action find_service_request_and_assignment`?